Live-Snap/Live-Snap/Controllers/ExportWallpaperViewController.swift

```diff
@@ -101,10 +101,12 @@ class ExportWallpaperViewController: UIViewController, PHLivePhotoViewDelegate {
     }
     
     func livePhotoView(_ livePhotoView: PHLivePhotoView, didEndPlaybackWith playbackStyle: PHLivePhotoViewPlaybackStyle) {
-        view.addSubview(toolBar)
-        view.addSubview(forceTouchNotifierLabel)
-        animateToolBarAndLivePhotoView()
-        view.isUserInteractionEnabled = true
+        if !toolBar.isDescendant(of: view) {
```

Comment:
view.subviews.contains(toolBar) or something similar would be more clear I think